.changeset/new-bags-trade.md

```diff
@@ -0,0 +1,15 @@
+---
+'sku': minor
+---
+
+Add `excludeFromBabel` option
```

Comment:
Feel like this should be called `excludePackagesFromBabel` to be more explicit about what it's targting.